modern-phytomorphology.csl

```diff
@@ -0,0 +1,182 @@
+      <name form="short" font-variant="small-caps" and="symbol" initialize-with="."/>
+      <et-al font-style="italic" font-variant="normal"/>
+      <substitute>
+        <names variable="editor" font-variant="normal"/>
+        <names variable="translator"/>
+        <choose>
+          <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+            <text variable="title" form="short" font-style="italic"/>
+          </if>
+          <else>
+            <text variable="title" form="short" quotes="false" font-variant="small-caps"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="article-newspaper article-magazine article-journal article book chapter dataset entry entry-dictionary entry-encyclopedia manuscript paper-conference personal_communication report review review-book speech thesis" match="any">
+        <text variable="title" text-case="capitalize-first" font-style="normal"/>
+      </if>
+      <else>
+        <text variable="title" text-case="title" font-variant="small-caps" font-weight="bold" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=", " prefix=". ">
+      <text variable="publisher"/>
+      <text variable="publisher-place"/>
+    </group>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="page" match="none">
+        <choose>
+          <if variable="DOI">
+            <text variable="DOI" prefix="https://doi.org/"/>
+          </if>
+          <else-if variable="URL">
+            <text variable="URL"/>
+            <choose>
+              <if variable="issued" match="none">
+                <date variable="accessed" prefix=". Accessed: " suffix=".">
+                  <date-part name="month" suffix=" "/>
+                  <date-part name="day" suffix=", "/>
+                  <date-part name="year"/>
+                </date>
+              </if>
+            </choose>
+          </else-if>
+          <else-if variable="issued" match="any">
+            <text variable="status" text-case="capitalize-first"/>
+          </else-if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <citation collapse="year-suffix" et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" year-suffix-delimiter=",">
+    <sort>
+      <key variable="issued"/>
+      <key variable="author"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group font-variant="small-caps" delimiter=" ">
+        <text macro="author-short" font-variant="normal"/>
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+        <group prefix=": ">
+          <text variable="locator" prefix=" "/>
+        </group>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-min="19" et-al-use-first="19" et-al-subsequent-min="19" et-al-subsequent-use-first="19">
```

Comment:
@novikoffav, `et-al-subsequent-min="19" et-al-subsequent-use-first="19"` doesn't have any effect since the values are the same as for `et-al-min="19" et-al-use-first="19"`. Can you delete these attributes?